tests/foreman/cli/test_repository.py

```diff
@@ -714,21 +723,22 @@ def test_positive_synchronize_file_repo(self, repo_options, repo, target_sat):
     @pytest.mark.tier2
     @pytest.mark.parametrize(
         'repo_options',
-        **parametrized(
-            [
-                (
-                    {
-                        'content-type': 'yum',
-                        'url': FAKE_5_YUM_REPO,
-                        'upstream-username': creds['login'],
-                        'upstream-password': creds['pass'],
-                    }
-                    for creds in valid_http_credentials()
-                    if not creds['http_valid']
-                )
-            ]
-        ),
+        [
+            {
+                'content-type': 'yum',
+                'url': FAKE_5_YUM_REPO,
+                'upstream-username': creds['login'],
+                'upstream-password': creds['pass'],
+            }
+            for creds in valid_http_credentials()
+            if not creds['http_valid']
+            and len(creds['login']) < 255
+            and not (not creds['login'] and creds['pass'])
+            and can_encode_latin(creds['login'])
+            and can_encode_latin(creds['pass'])
```

Comment:
@dosas if this amount of alterations is needed, I'd rather consider looking at the valid_http_credentials() function itself. Maybe creating something like `'yum_compatible'` parameter there would be enough to isolate the valid combinations and make them reusable in other yum-releated tests by using just `if creds['yum_compatible']`

Reply:
also fine with me